packages/core/src/components/TextField/TextField.tsx

```diff
@@ -72,6 +73,11 @@ export const TextField: FC<Props> & WithStyle = React.memo(
             mask && value && value.toString().length === mask.length && setMaskLabel(value.toString());
         }, [value, mask]);
 
+        useEffect(() => {
+            const inputWidth = inputRef.current.offsetWidth;
```

Comment:
You don't need to declare a variable over here. you can directly write it like this `setInputWidth(inputRef.current.offsetWidth);`

Reply:
Updated: 086277f